<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>用户界面</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTabWidget" name="tabWidget">
            <property name="tabPosition">
             <enum>QTabWidget::TabPosition::North</enum>
            </property>
            <property name="tabShape">
             <enum>QTabWidget::TabShape::Rounded</enum>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <property name="elideMode">
             <enum>Qt::TextElideMode::ElideNone</enum>
            </property>
            <widget class="QWidget" name="tab_0">
             <attribute name="title">
              <string>盖浇饭</string>
             </attribute>
            </widget>
            <widget class="QWidget" name="tab_1">
             <attribute name="title">
              <string>凉菜</string>
             </attribute>
            </widget>
            <widget class="QWidget" name="tab_2">
             <attribute name="title">
              <string>荤菜</string>
             </attribute>
            </widget>
            <widget class="QWidget" name="tab_3">
             <attribute name="title">
              <string>素菜</string>
             </attribute>
            </widget>
            <widget class="QWidget" name="tab_4">
             <attribute name="title">
              <string>甜点</string>
             </attribute>
            </widget>
            <widget class="QWidget" name="tab_5">
             <attribute name="title">
              <string>饮品</string>
             </attribute>
            </widget>
            <widget class="QWidget" name="tab_6">
             <attribute name="title">
              <string>其他</string>
             </attribute>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_0">
            <property name="text">
             <string>加入购物车</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>总价：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QListView" name="listView"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_1">
            <property name="text">
             <string>从购物车移除</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>结算</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>管理员界面</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTabWidget" name="tabWidget_2">
            <property name="tabPosition">
             <enum>QTabWidget::TabPosition::North</enum>
            </property>
            <property name="tabShape">
             <enum>QTabWidget::TabShape::Rounded</enum>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <property name="elideMode">
             <enum>Qt::TextElideMode::ElideNone</enum>
            </property>
            <widget class="QWidget" name="tab_7">
             <attribute name="title">
              <string>盖浇饭</string>
             </attribute>
            </widget>
            <widget class="QWidget" name="tab_8">
             <attribute name="title">
              <string>凉菜</string>
             </attribute>
            </widget>
            <widget class="QWidget" name="tab_9">
             <attribute name="title">
              <string>荤菜</string>
             </attribute>
            </widget>
            <widget class="QWidget" name="tab_10">
             <attribute name="title">
              <string>素菜</string>
             </attribute>
            </widget>
            <widget class="QWidget" name="tab_11">
             <attribute name="title">
              <string>甜点</string>
             </attribute>
            </widget>
            <widget class="QWidget" name="tab_12">
             <attribute name="title">
              <string>饮品</string>
             </attribute>
            </widget>
            <widget class="QWidget" name="tab_13">
             <attribute name="title">
              <string>其他</string>
             </attribute>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>添加菜品</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="text">
             <string>删除菜品</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QListView" name="listView_2"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_4">
            <property name="text">
             <string>从订单中移除</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>831</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menud">
    <property name="title">
     <string>管理员登入</string>
    </property>
    <addaction name="actionlog"/>
    <addaction name="actionexit"/>
   </widget>
   <addaction name="menud"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionlog">
   <property name="text">
    <string>log</string>
   </property>
  </action>
  <action name="actionexit">
   <property name="text">
    <string>exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
